app/db/handlers.py

```diff
@@ -85,7 +85,11 @@ async def create(self, obj: M) -> M:
     async def get(
         self, id: str, extra_conditions: list[ColumnExpressionArgument] | None = None
     ) -> M:
-        extra_conditions = extra_conditions or self.default_extra_conditions
+        extra_conditions = extra_conditions or []
```

Comment:
I think this changes must be discussed first. The original idea (probably wrong but intentional) was that the `extra_conditions` overrides the default ones not extend them.

Reply:
Finally I removed such default_extra_conditions, I think it's not clear. Please rebase your PR on latest main. Thank you!